### PR TITLE
Displayed values extension

### PR DESCRIPTION
**Pull request by mtwaro**
- Add total coin supply
- Add active wallets number
- Add market cap
- Add avg tx volume in the last 24h
- Add exchange rates

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -245,8 +245,24 @@ app.runOnStartup = function() {
 		utils.refreshExchangeRates();
 	}
 
+	if (global.totalCoinSupply == null) {
```
Comment (kj-crypto) [inline app.js:248]: Isn't safer to use `===` (strict comparison) rather than `==` ?

File: app/coins/btc.js [hunk 2]
```diff
@@ -226,25 +226,25 @@ module.exports = {
...
+			}
+			return null;
+		}
+	},
+	exchangeRateDataBTC:{
+		jsonUrl:"https://api.liquid.com/products/619",
+		responseBodySelectorFunction:function(responseBody) {
+			//console.log("Exchange Rate Response: " + JSON.stringify(responseBody));
+			if (responseBody.last_traded_price) {
+				let exchangeRate = responseBody.last_traded_price;
```
Comment (kj-crypto) [inline app/coins/btc.js:245]: What about just returning `return responseBody.last_traded_price;` without setting temporary variable?

File: app.js (hunk 1 above)
Comment (mtwaro) [reply app.js:248]: Non-strict comparison covers all false-like values, e.g.:
```
null
undefined
NaN
empty string ("")
false
```
And that's what we want to achieve. I do agree however, that writing `value == null` is not too elegant, so I'll change it to `!value`

File: app/coins/btc.js (hunk 2 above)
Comment (mtwaro) [reply app/coins/btc.js:245]: Will be done.

**Pull request merged**